en/publish/publish-bytedance.md

```diff
@@ -0,0 +1,122 @@
+
+- **Code Directory for Open Data Domains**
+
+  This item is optional and is used to access the open data domain of ByteDance , please refer to [Access the Open Data Domain of ByteDance Mini Game](publish-bytedance-sub-domain.md) for the specific publishing process.
+
+### Run Preview
+
+- Click the **Open** button behind **Build Path** after the build is finished, you can see that the ByteDance mini game project folder **bytedance** is generated under the build directory of the publish package, which already contains the configuration files of bytedance  environment: `game.json` and `project. config.json`.
+
+  ![](./publish-bytedancegame/package.png)
+
+- Use the **ByteDance developer tool** to open the **bytedance** folder generated by the build, you can open the ByteDance game project and preview the debugging game content. Please refer to [Introduction to ByteDance Developer Tools](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/developer-instrument/development-) for details on how to use the developer tools. assistance/mini-app-developer-instrument).
+
+  ![](./publish-bytedancegame/preview.png)
+
+## Resource Management for ByteDance Mini Game Environment
+
+ByteDance mini games are similar to WeChat mini games in that there is a package size limit, and additional assets over 4MB must be downloaded via network requests.
+
+Cocos Creator has already done the downloading, caching and versioning of remote assets for developers. The specific implementation logic and operation steps are similar to those of WeChat mini games, please refer to [WeChat Mini Game Resource Management](./publish-wechatgame.md#resource-management-for-the-wechat-mini-games).
+
+## Subpackage Loading
+
+Cocos Creator supports ByteDance mini game subpackage function since **v2.4.4**, and the usage is similar to WeChat mini game. Please refer to [Mini Game Subpackage](./subpackage.md).
+
+ByteDance  needs specific version to support subpackage function. The version requirements of ByteDance product are as follows:
+
+| product | android | iOS |
+| :-- | :--- | :--- |
+| Douyin | v13.6.0 | v13.7.0 |
+| Toutiao | v7.9.9 | v7.9.8 |
+
+ByteDance Developer Tools requires a version higher than or equal to **2.0.6** and less than **3.0.0**. The debugging base library requires 1.88.0 and above.
+
+> **Note**: if the product version does not support supackage loading, the engine will load the subpackage as a normal asset bundle.
+
+There are currently the following limits on the size of the mini game subpackages:
+- The size of all subpackages of the whole mini game should not exceed 16 MB
+- The size of a single subpackage/main package cannot exceed 4 MB
+
+For details, please refer to the official documentation of [ByteDance Game Subpackage Loading](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/framework/subpackages/introduction)
```

Comment:
For details, please refer to the official documentation of [ByteDance Game Subpackage Loading](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/framework/subpackages/introduction)
-->
For details, please refer to the official [ByteDance Game Subpackage Loading](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/framework/subpackages/introduction) documentation.